documentation/documentation/docs/projectform/project_upload_form.md

```diff
@@ -0,0 +1,33 @@
+# Project upload guide
+A teacher can upload a new project of his course by going to the url ```projects/create``` where he will need to fill in data corresponding to the project he wants to upload.
+![Projectform](/img/project_form_1.png)
+
+The first 3 fields are rather straightforward, you just need to fill in the title you want for the project, the description and select the course you want the project for. The course selection is in a dropdown where you see all the courses you can see all the courses you are teacher of.
+
+![Projectform](/img/project_form_2.png)
+
+Next theres a calender where you need to choose deadlines of the project, every deadline can have a deadline and a timestamp.
+As seen in the image above you'll see a table of every deadline you added, if you want to remove a deadline simply click on the calender date itself and click the remove icon.
+
+![Projectform](/img/project_upload_form_3.png)
+
+Then there is a checkbox that contains the value visible for students, this dictated if a project should be immediately visible on upload or not.
+
+![Projectform](/img/project_upload_form_4.png)
+
+To upload assignment files you can use the given file uploader, it must be a .zip file.
```

Comment:
To upload assignment files you can use the given file uploader, it must be a .zip file. -> To upload assignment files you can use the given file uploader. Files must be uploaded as a .zip file.

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078